examples/passport-oauth2-login/client/web-app.js

```diff
@@ -0,0 +1,93 @@
+// Copyright IBM Corp. 2014,2015. All Rights Reserved.
+// Node module: loopback-example-passport
```

Comment:
The module name does not match package.json.

Reply:
agreed